lua/lscs/autorun/cl_forcepowers.lua

```diff
@@ -278,6 +279,7 @@ local function PlayerButtonUp( ply, button )
 	for _, name in pairs( Input ) do
 		for ID, power in pairs( ForcePowers ) do
 			if power.item.id == name then
+				if hook.Run( "LSCS:PlayerCanForce", ply, power ) == false then continue end
```

Comment:
maybe always allow stopuse, because if permissions are revoked during forcepower use, it will never stop the forcepower